addons/stock_picking_batch/views/stock_picking_batch_views.xml

```diff
@@ -218,14 +218,25 @@
         <field name="context">{'search_default_draft': True, 'search_default_in_progress': True}</field>
         <field name="search_view_id" ref="stock_picking_batch_filter"/>
         <field name="help" type="html">
-          <p class="o_view_nocontent_smiling_face">
-            Create a new batch transfer
-          </p><p>
-            The goal of the batch transfer is to group operations that may
-            (needs to) be done together in order to increase their efficiency.
-            It may also be useful to assign jobs (one person = one batch) or
-            help the timing management of operations (tasks to be done at 1pm).
-          </p>
+            <div class="container mt-5">
+                <div class="row g-5">
+                    <div class="col-lg-4 opaque-column">
+                        <img src="/stock_picking_batch/static/shapes/wave-picking.svg" class="img-thumbnail w-100 p-4 mb-4" alt="" loading="lazy" style=""/>
+                        <h5>Wave picking</h5>
+                        <p class="small">Launch picking orders per aisle, or area, and regroup at packing zone. Ideal for large warehouses.</p>
+                    </div>
+                    <div class="col-lg-4">
+                        <img src="/stock_picking_batch/static/shapes/batch-picking.svg" class="img-thumbnail w-100 p-4 mb-4" alt="" loading="lazy" style=""/>
+                        <h5>Batch picking</h5>
+                        <p class="small">Regroup multiple orders in one picking, and consolidate at the packing zone.</p>
+                    </div>
+                    <div class="col-lg-4">
+                        <img src="/stock_picking_batch/static/shapes/cluster-picking.svg" class="img-thumbnail w-100 p-4 mb-4" alt="" loading="lazy" style=""/>
+                        <h5>Cluster picking</h5>
+                        <p class="small">Pick multiple orders in one trip, and prepare orders as you pick, to save time at packing. Ideal for small products.</p>
+                    </div>
+                </div>
+            </div>
         </field>
```

Comment:
I'm not crazy with how the wording of this is inconsistent from the wording in the backend (we always say "Transfer" instead of "Picking" and we never refer to "Picking orders" except in the documentation/odoo.com site). I think it may be confusing to users. It's ok for this part of the review process, but it should be reconsidered for the final review before merging.

Also, I know it's copy paste from the website, but some of the English grammar is wrong/clunky so we should maybe consider fixing it here (and I can bug the marketing guys to fix it on the website)

cc @thomasdeleval